<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemorandumDialog</class>
 <widget class="QDialog" name="MemorandumDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>备忘信息窗口 - TDCS/CTC3.0 通用车站操作终端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 210, 218);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>9</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>建立时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>44</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
QPushButton{text-align: left;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>79</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>过期时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>得知</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>9</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>9</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>79</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>79</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d HH:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>44</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>标志</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>479</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
   </property>
   <property name="text">
    <string>备忘</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>114</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>79</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>生效时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>用户查看</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>114</y>
     <width>441</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
QPushButton{text-align: left;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(214, 227, 235);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(187, 197, 204);</string>
         </property>
         <property name="text">
          <string>序号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(187, 197, 204);</string>
         </property>
         <property name="text">
          <string>标题</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(187, 197, 204);</string>
         </property>
         <property name="text">
          <string>登记时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>309</y>
     <width>471</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 227, 235);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 227, 235);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
